pegasus/sites.v3/code.org/views/breadcrumbs.haml

```diff
@@ -0,0 +1,13 @@
+%div.breadcrumbs
+  - trail.each_with_index do |crumb, index|
+    - if !crumb[:url] || index == trail.size - 1
+      %strong.crumb.active
+        #{crumb[:text]}
+    - else
+      %a.crumb.inactive{href:"#{crumb[:url]}"}
+        #{crumb[:text]}
+      >
+:css
+  .active {
```

Comment:
Might want to namespace this to make this a bit more specific, i.e., if something else on the page uses the class "active" it might then turn orange. `.crumb.active` would seem specific enough